dace/runtime/include/dace/cuda/copy.cuh

```diff
@@ -24,11 +24,17 @@
 #ifndef __DACE_CUDACOPY_CUH
 #define __DACE_CUDACOPY_CUH
 
+#ifdef __CUDACC__
 #include <cuda_runtime.h>
+#elif defined(__HIPCC__)
+#include <hip/hip_runtime.h>
```

Comment:
Oh what... I guess the plan is to eventually rename all of this to GPU then?

Reply:
The filenames, but not the code